tree.py

```diff
@@ -0,0 +1,64 @@
+from shiny import reactive, ui, render, module
+import scHPL
+from typing import List
+from shiny.render.renderer import Jsonifiable, Renderer
+from htmltools import HTMLDependency
+
+tree_dep = HTMLDependency(
+    "tree",
+    "1.0.0",
+    source={"subdir": "tree"},
+    script={"src": "tree.js", "type": "module"},
+    all_files=True
+)
+
+def output_tree(id):
+    return ui.div(
+        ui.HTML("<script src='//unpkg.com/3d-force-graph'></script>"),
+        tree_dep,
+        id=module.resolve_id(id),
+        class_="shiny-tree-output"
+    )
+
+class render_tree(Renderer[dict]):
+    def auto_output_ui(self):
+        return ui.ouptut_tree(self.ouptut_name)
```

Comment:
spelling mistakes: ouptut_trees and ouptput_name

Reply:
Oopsie